### DISCUSSION TITLE
Are there any airsonic/subsonic Android client developers out there who are interested in developing a client for AAOS?

### DISCUSSION THREAD

**Discussion by litebito** [kagemomiji/airsonic-advanced#623] [General]
Hi,

I've been looking for a long time already, for a client for AAOS (Android Automotive OS) which can connect to a (self)hosted Airsonic/Subsonic/Navidrome. Is anyone aware of a project or work being done for such a client?

Or, does anyone want to get started on this?

**Comment by NoBurpees** [comment]
Checkout this:

https://github.com/CappielloAntonio/tempo

Found it while looking for subsonic clients and it states Android Auto support - haven't tried it myself, though

**Reply by luxdi** [reply]
^ Android Auto isn't the same thing as AAOS. To the OP, I'm not aware of any apps that run on AAOS that can connect to Subsonic etc. Though as AAOS is open source I'm surprised there isn't more of anything available really.

**Comment by litebito** [comment]
As far as I understood, most devs seem to avoid it, because of the over the top administrative strain and stress to get things approved by Google and car manufacturers before it's available in the play stores.
It can take up to six months before you get anywhere with Google on AAOS apparently .

**Reply by luxdi** [reply]
I've just seen this at the Jellyfin forums. I haven't got AAOS to try out but may be it could work with Airsonic.

https://github.com/bendardenne/sharkmarmalade/

**Reply by litebito** [reply]
Thanks for the note.
I checked with the dev, it won't work with the subsonic API, only with Jellyfin.

**Comment by litebito** [comment]
Just in case others are looking...
On Android I moved from the free opensource substreamer to https://symfonium.app which in its extensive features set has Android Auto support.

I know it is not AAOS, but it is something, Lately I find myself switching to Android Auto a lot, because of lack of features and apps in AAOS.

Most devs I've asked, don't want to bother with developing apps for AAOS because of how difficult it is made by Google and car manufacturers to get an app published. It can take 6 months or even longer before an AAOS app is reviewed and published.

It is not free, but worth its price. for me it has been proven super stable, no crashes no hickups and has an amazing featureset. It even plays the tracks substreamer could not play. Also has an extensive list of options for sources (besides the subsonic API)
(I'm not affiliated)

In the mean time, I'm keeping my eyes open for development on an native AAOS client.
